### PR TITLE
US-6.5: Implement my settings portlet

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/groovy/platformNavigation/portlet/UIUserPlatformToolBarPortlet/UIUserPlatformToolBarPortlet.gtmpl [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<%
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/groovy/platformNavigation/portlet/UIUserPlatformToolBarPortlet/UIUserPlatformToolBarPortlet.gtmpl]: Not needed, we should remove this portlet from topbar

File: digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue [hunk 2]
```diff
@@ -3,7 +3,66 @@
...
+  export default {
+    methods : {
+      logOut() {
+        eXo.portal.logout();
+      },
+      changeLanguage() {
+        if(document.getElementById('UIMaskWorkspace')) {ajaxGet(eXo.env.server.createPortalURL('UIPortal', 'ChangeLanguage', true));}
+      },
+      accountSettings() {
+        if(document.getElementById('UIMaskWorkspace')) {ajaxGet(eXo.env.server.createPortalURL('UIPortal', 'AccountSettings', true));}
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue]: ajaxGet should be removed I think

Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/vue-app/sidebar/mySettings/components/MySettingsApp.vue]: Done!

File: digital-workplace-webapps/src/main/webapp/groovy/platformNavigation/portlet/UIUserPlatformToolBarPortlet/UIUserPlatformToolBarPortlet.gtmpl (hunk 1 above)
Comment (SaraBoutej) [reply digital-workplace-webapps/src/main/webapp/groovy/platformNavigation/portlet/UIUserPlatformToolBarPortlet/UIUserPlatformToolBarPortlet.gtmpl]: Done!

**Pull request merged**